src/common/snippets/src/pass/tokenization.cpp

```diff
@@ -81,9 +82,15 @@ bool SnippetsTokenization::run_on_model(const std::shared_ptr<ov::Model>& m) {
 
     manager.register_pass<EnumerateNodes>();
     manager.register_pass<ExtractReshapesFromMHA>();
+    // This pass mustn't be registered in GraphRewrite with other tokenization passes
+    // since it changes the nodes after the matched root node
```

Comment:
Just wondering (personal interest in expanding my horizons): what does "it changes the nodes after the matched root node" mean? 🤔 Why `TokenizeSnippets` doesn't change the nodes?
Thank you in advance!

Reply:
The comment is updated

Reply:
> MatcherPass doesn't change nodes below the matched root.

Oh exactly! I forgot that in MHA tokenization pass we match on first `MatMul`. Thank you very much for the explanation!